kotlin/android/app/build.gradle

```
@@ -30,7 +30,7 @@ android {
 
             // Debug Config
             manifestPlaceholders.hostName = "app.firez.one"
-            buildConfigField("String", "AUTH_DEST", "\"app.firez.one\"")
+            buildConfigField("String", "FIREZONE_HOST", "\"app.firez.one\"")
```

For testing locally with the Docker environment, it should be "localhost":

```suggestion
            buildConfigField("String", "AUTH_HOST", "\"localhost\"")
            buildConfigField("String", "AUTH_SCHEME", "\"http\"")
            buildConfigField("String", "AUTH_PORT", "8080")
            buildConfigField("String", "CONTROL_PLANE_URL", "\"ws://localhost:8081/\"")
```

We also need scheme, port, and control plane url.

---

You can spin up a local portal with Docker using the guide here: https://github.com/firezone/firezone/blob/main/elixir/README.md

Note the web (auth) port is 8080 and the api (connlib control) port is 8081

---

^ This can replace PortalMock :-)